.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "repos/jikesrvm"]
 	path = repos/jikesrvm
 	url = https://github.com/mmtk/jikesrvm
+	branch = cleanup_before_upstream
```

Comment:
What is your plan here, keep using the branch or merging the branch and then fixing this?

Reply:
merging the branch and then fixing this